cfy_manager/components/restservice/config/authorization.conf

```diff
@@ -68,6 +68,9 @@ permissions:
 # The roles here refer to both system-roles and tenant-roles (for the
 # relevant tenant only).
 #############################################################################
+  getting_started_modal
```

Comment:
I'm wondering about the permission name - `_modal` is kind of implementation detail, I'm not sure it should be part of the name. What if the implementation changed from modal to a page?
I'd suggest just `getting_started` for the permission name.

Reply:
changed to getting_started